templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
```

Comment:
This link currently doesn't lead anywhere. You might want to replace this with an actual URL when you implement the 'Manufacturers' feature.